modules/ossm-migrating-read-me-sidecar-injection-considerations.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// * service-mesh-docs-main/about/ossm-migrating-assembly.adoc
+
+//Start of an overall Migrating section.
+//Section is most likely to be reworked/restructured with OSSM 2 to OSSM 3 migration guides for GA. Unknown how many migration guides there are at this time (11/11/2024). It would be beneficial to be able to link from differences to the relevent migration guide so that users A) understand the change, esp significant changes like new operator, installing tracing and Kiali separately, gateways, etc.
+
+:_mod-docs-content-type: CONCEPT
+[id="ossm-migrating-read-me-sidecar-injection-considerations_{context}"]
+= New considerations for sidecar injection
+
+{SMProductName} 2 supported using pod annotations and labels to configure sidecar injection and there was no need to indicate which control plane a workload belonged to.
+
+With {SMProduct} 3, even though the Istio control plane discovers a namespace, the workloads present in that namespace still require sidecar proxies to be included as workloads in the service mesh, and to be able to use Istio's many features.
+
+In {SMProduct} 3, sidecar injection works the same way as it does for Istio, with pod or namespace labels used to trigger sidecar injection. However, it might be necessary to include a label that indicates which control plane the workload belongs to.
+
+[NOTE]
+====
+The Istio Project has deprecated pod annotations in favor of labels for sidecar injection.
+====
+
+When an `Istio` resource has the name `default` and `InPlace` upgrades are used, there is a single `IstioRevision` with the name `default` and the label `istio-injection=enabled` for sidecar injection.
+
+However, an `IstioRevision` resource is required to have a different name in the following cases:
+
+* Multiple control plane instances are present.
+* A `RevisionBased`, canary-style control plane upgrade is in progress.
+
+If there are running multiple control plane instances, or you chose the `RevisionBased` update strategy during your {SMProduct} 3 installation, then an `IstioRevision` resource is required to have a different name than `default`. When that happens, it is necessary to use a label that indicates which control plane revision the workload(s) belong to by specifying `istio.io/rev=<istiorevision_name>`.
```

Comment:
Also, per the [ISG](https://www.ibm.com/docs/en/ibm-style?topic=punctuation-parentheses#plurals) we shouldn't use parentheses to indicate potential plurals:
```suggestion
If there are running multiple control plane instances, or you chose the `RevisionBased` update strategy during your {SMProduct} 3 installation, then an `IstioRevision` resource is required to have a different name than `default`. When that happens, it is necessary to use a label that indicates which control plane revision the workloads belong to by specifying `istio.io/rev=<istiorevision_name>`.
```
If you had to choose one and singular "workload" feels better, feel free to choose that option instead